grobid-trainer/resources/dataset/citation/corpus/crossref_raw_citations.xml

```diff
@@ -0,0 +1,54 @@
+<bibl> <author>Pearson, K.</author> <date>1901</date>. <title level="a">On lines and planes of closest fit to systems of points in space</title>, <title level="j">The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</title>, <biblScope type="vol">2</biblScope>(<biblScope type="issue">11</biblScope>), <biblScope type="pp">559--572</biblScope>. DOI=<idno type="DOI">10.1080/14786440109462720</idno>. </bibl>
+<bibl> <ptr type="web">http://www.icmje.org/icmje-recommendations.pdf</ptr> [Consulté le <date>2 janvier 2016</date>]. </bibl>
+<bibl> <author>Ben Taskar Pieter Abbeel and Daphne Koller</author>. <date>2012</date>. <title level="a">Discriminative Probabilistic Models for Relational Data</title>. <idno type="arXiv">arxiv:cs.LG/1301.0604</idno> </bibl>
+<bibl> <author>Meng Qu and Jian Tang</author>. <date>2019</date>. <title level="a">Probabilistic Logic Neural Networks for Reasoning</title>. <idno type="arXiv">arxiv:cs.LG/1906.08495</idno> </bibl>
+<bibl> <author>Bishan Yang Scott Wen-tau Yih Xiaodong He Jianfeng Gao and Li Deng</author>. <date>2015</date>. <title level="a">Embedding Entities and Relations for Learning and Inference in Knowledge Bases</title>. In <title level="m">Proceedings of the International Conference on Learning Representations (ICLR)</title> <date>2015</date>. <ptr type="web">https://www.microsoft.com/en-us/research/publication/embedding-entities-and-relations-for-learning-and-inference-in-knowledge-bases/</ptr> </bibl>
+<bibl> <author>Christoph Molnar</author>. <date>2020</date>. <title level="m">Interpretable Machine Learning</title> .<publisher>Leanpub</publisher> <pubPlace>Victoria Canada</pubPlace>. <ptr type="web">https://christophm.github.io/interpretable-ml-book/</ptr>. </bibl>
+<bibl> <author>Günter Klambauer Thomas Unterthiner Andreas Mayr and Sepp Hochreiter</author>. <date>2017</date>. <title level="a">Self-Normalizing Neural Networks</title>. <idno type="arXiv">arxiv: 1706.02515</idno> [cs.LG] </bibl>
+<bibl> <author>Byung-Hak Kim Ethan Vizitei and Varun Ganapathi</author>. <date>2018</date>. <title level="a">GritNet: Student Performance Prediction with Deep Learning</title>. <idno type="arXiv">arxiv: 1804.07405</idno> [cs.LG] </bibl>
+<bibl> <author>Sercan O. Arik and Tomas Pfister</author>. <date>2019</date>. <title level="a">TabNet: Attentive Interpretable Tabular Learning</title>. <idno type="arXiv">arxiv: 1908.07442</idno> [cs.LG] </bibl>
+<bibl> <orgName>Wikipedia</orgName>. <date>2019</date>. <title level="m">PageRank</title>, Retrieved <date>September 5, 2019</date>, from <ptr type="web">https://en.wikipedia.org/wiki/PageRank</ptr> </bibl>
+<bibl> <author>Mohammad Al Hasan, Vineet Chaoji, Saeed Salem, and Mohammed Zaki</author>. [n.d.]. <title level="m">Link Prediction using Supervised Learning</title>. ([n. d.]), 10. </bibl>
```

Comment:
Checking the article, it appears that `10` is the number of pages. 
`<biblScope type="pp">10</biblScope>`